### ISSUE TITLE
Unnecessary assumption in sorted_take

### ISSUE THREAD

**Issue by ybertot** [CoqEAL/CoqEAL#40]
Hello,
in CoqEAL0.1, ` sorted_take` (from ssrcomplements.v) had no transitivity assumption.  In CoqEAL1.0.5, such an assumption exists.  It may turn out be a problem one day.

**Comment by proux01** [comment]
Indeed, this lemma should be removed as it is now in MC https://github.com/math-comp/math-comp/blob/14097e37cb3b0590ef5d91809acf4fed0fb22f1a/mathcomp/ssreflect/path.v#L369

**Comment by ybertot** [comment]
I do not understand.  `sorted_take` does not occur in the current master branch of math-comp.  What is the commit that @proux01 mentions attached to?

**Comment by proux01** [comment]
It's `take_sorted` : https://github.com/math-comp/math-comp/blob/master/mathcomp/ssreflect/path.v#L369

**Comment by ybertot** [comment]
thks, I did not pay enough attention.

**Comment by proux01** [comment]
I'd rather keep this open to remember to do the change once MC 1.13 is out.
